po/es.po

```diff
@@ -0,0 +1,144 @@
+# Spanish translations for com.github.elfenware.obliviate package.
+# Copyright (C) 2020 THE com.github.elfenware.obliviate'S COPYRIGHT HOLDER
+# This file is distributed under the same license as the com.github.elfenware.obliviate package.
+# Darshak Parikh, 2020.
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: com.github.elfenware.obliviate\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2020-12-18 20:37+0530\n"
+"PO-Revision-Date: 2020-12-18 20:37+0530\n"
+"Last-Translator: Darshak Parikh\n"
+"Language-Team: Español\n"
+"Language: es\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=2; plural=(n != 1);\n"
+
+#: src/MainView.vala:43 src/MainView.vala:160
+msgid "Copied to clipboard"
+msgstr "Copiado al portapapeles"
+
+#: src/MainView.vala:48
+msgid "Site:"
+msgstr "Sitio:"
+
+#: src/MainView.vala:54
+msgid "GitHub"
+msgstr "GitHub"
+
+#: src/MainView.vala:60
+msgid "Site is not case-sensitive. “GitHub” equals “github”."
+msgstr "El sitio no distingue entre mayúsculas y minúsculas. «GitHub» es igual a «github»."
+
+#: src/MainView.vala:63
+msgid "Cipher key:"
+msgstr "Clave de cifrado:"
+
+#: src/MainView.vala:73
+msgid "correct horse battery staple"
+msgstr "correcto caballo batería grapa"
+
+#: src/MainView.vala:81
+msgid "Show or hide the cipher key"
+msgstr "Mostrar u ocultar la clave de cifrado"
+
+#: src/MainView.vala:87
+msgid "Derive Password"
+msgstr "Derivar Contraseña"
```

Comment:
I understand the word "derive" because I've doing crypto at work for (way too many) years, but I feel like this is crypto technical vocabulary.
Personally, I'll use "Generate Password"/"Generar contraseña" here.

Reply:
That's what I was going to go with, but like I said on Twitter, “derive” makes it clearer that it's based on the two inputs.